src/AppInstallerCommonCore/Public/winget/ManifestCommon.h

```diff
@@ -370,6 +379,8 @@ namespace AppInstaller::Manifest
 
     IconResolutionEnum ConvertToIconResolutionEnum(std::string_view in);
 
+    RepairBehaviorEnum ConvertToRepairBehaviorEnum(const std::string& in);

```

Comment:
```suggestion
    RepairBehaviorEnum ConvertToRepairBehaviorEnum(std::string_view in);

```
For new code if possible